vm/neovm/executor.go

```diff
@@ -200,7 +206,10 @@ func (self *Executor) ExecuteOp(opcode OpCode, context *ExecutionContext) (VMSta
 		if target < 0 || target >= int64(len(self.Context.Code)) {
 			return FAULT, errors.ERR_DCALL_OFFSET_ERROR
 		}
-		self.Context.SetInstructionPointer(target)
+		err := self.Context.SetInstructionPointer(target)
+		if err != nil {
+			return FAULT, err
+		}
 	case RET:
 		self.Context, _ = self.PopContext()
```

Comment:
there is maybe error

Reply:
this error should be omitted, I have added a note for this.